### PR TITLE
add nnfx theme

### PR DESCRIPTION
**Pull request by RocketMan**
This PR adds a new theme nnfx, inspired by view-source in the Firefox browser.

Screenshot:
Screenshot from 2020-05-18 18-19-24

### CODE REVIEW

**Comment by yyyc514** [comment]
Please attach a CSS and markdown snap as well so we can see all the different classes in action.

**Comment by yyyc514** [comment]
>  inspired by view-source in the Firefox browser.

I was just about to say it looked familiar... LOL.

File: src/styles/nnfx.css [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+ * nnfx - a theme inspired by Netscape Navigator/Firefox
+ *
+ * @version 1.0.0
+ * @author (c) 2020 Jim Mason <jmason@ibinx.com>
+ * @license https://creativecommons.org/licenses/by-sa/4.0  CC BY-SA 4.0
+ */
+
+.hljs {
+  display: block;
+  overflow-x: auto
```
Comment (yyyc514) [inline src/styles/nnfx.css:11]: Missing `;`?

Comment (RocketMan) [reply src/styles/nnfx.css]: Sorry about that; good catch!

**Comment by RocketMan** [comment]
> Please attach a CSS and markdown snap as well so we can see all the different classes in action.

I've pushed up a browser build with the theme here:

https://stage.ibinx.com/nnfx_build/demo/

Please let me know if there is a preferred way to attach directly to the PR in github.

**Comment by yyyc514** [comment]
@RocketMan Thanks!

**Pull request merged**